modules/RTC/ScreenObtainer.js

```diff
@@ -116,13 +116,24 @@ const ScreenObtainer = {
                     });
             };
         } else if (RTCBrowserType.isElectron()) {
-            obtainDesktopStream = (_, onSuccess, onFailure) =>
-                window.JitsiMeetElectron.obtainDesktopStream(
+            obtainDesktopStream = (_, onSuccess, onFailure) => {
+
+                let obtainStreamId
+                    = window.JitsiMeetElectron.obtainDesktopStream;
```

Comment:
I've removed the call to obtainDesktopStream and I added a couple electron specific error messages.